test/ao/sparsity/test_activation_sparsifier.py

```diff
@@ -78,6 +123,79 @@ def check_constructor(self, activation_sparsifier, model, defaults, sparse_confi
             # all the keys in combined_defaults should be present in sparsifier defaults
             assert config == combined_defaults.get(key, None)
 
+    def check_register_layer(self, activation_sparsifier, defaults, sparse_config, layer_args_list):
```

Comment:
Again, it looks like the checks are within a generic testcase class, but they are only applicable to your test class. Would be worthhile merging them

Reply:
done